main.qml

```diff
@@ -2,7 +2,7 @@ import QtQuick 2.7
 import QtWebEngine 1.4
 import QtWebChannel 1.0
 import QtQuick.Window 2.2 // for Window instead of ApplicationWindow; also for Screen
-import QtQuick.Controls 2.3 // for ApplicationWindow
+import QtQuick.Controls 1.4 // for ApplicationWindow
```

Comment:
are you sure there wasn't a reason for upgrading this to 2.3 in the first place?

Reply:
The last change is here https://github.com/Stremio/stremio-shell/commit/87afa21662f6e05d5512c3c414919f8588680d29 It doesn't look like there is actual need for this change. And I couldn't build it on Ubuntu with QtQuick.Controls 2.3 with 1.4 it builds on every platform.